src/common/modules/navigations/gnb/modules/gnb-menu/modules/dashboard-recent-favorite/modules/GNBDashboardMenu.vue

```diff
@@ -22,7 +24,9 @@
                                         :is-new="subMenu.isNew"
                         />
                     </div>
-                    <div class="gradient-box" />
+                    <div v-if="isOverflown"
+                         class="gradient-box"
+                    />
```

Comment:
question: Why gradient-box div is in GNBDashboardMenu.vue?

Reply:
Because I need to implement the same UI in refreshTab.